request_logging.go

```diff
@@ -61,17 +61,17 @@ func LoggingMiddleWare(h http.HandlerFunc) http.HandlerFunc {
 					http.Error(lw, fmt.Sprintf("%s \n%s", msg, stack), http.StatusInternalServerError)
 				}
 				fields["status_text"] = http.StatusText(lw.status)
-				switch {
-				case lw.status < 300:
-					if debug {
+				if debug {
```

Comment:
I would prefer dropping it, but I agree, if someone relies on this it is better this way